TUnit.Assertions/Extensions/Is/Strings/StringIsExtensions.cs

```diff
@@ -13,7 +13,10 @@ public static class StringIsExtensions
 {
     public static StringEqualToAssertionBuilderWrapper IsEqualTo(this IValueSource<string> valueSource, string expected, [CallerArgumentExpression("expected")] string doNotPopulateThisValue1 = "")
     {
-        return IsEqualTo(valueSource, expected, StringComparison.Ordinal, doNotPopulateThisValue1);
+        var assertionBuilder = valueSource.RegisterAssertion(new StringEqualsExpectedValueAssertCondition(expected, StringComparison.Ordinal)
```

Comment:
Could we still call the overload but pass in an empty expression?

Reply:
I tried it, but both, with null or an empty string, the result contained it in the generated expression string, so I had to revert this commit.

Reply:
Can you try again with the latest main merged in? I believe I've sorted that

Reply:
I merged in `main` and it seems to work when I provide `null`.